test/index.js

```diff
@@ -189,7 +189,7 @@ test('Setoid', function() {
   eq(Z.Setoid.test(''), true);
   eq(Z.Setoid.test([]), true);
   eq(Z.Setoid.test({}), true);
-  eq(Z.Setoid.test({'@@type': 'my-package/Quux'}), false);
+  eq(Z.Setoid.test({'@@type': 'my-package/Quux'}), true);
```

Comment:
That's right. `{'@@type': 'my-package/Quux'}` is considered the prototype for the Quux type, so its `@@type` property is ignored.